pkg/components/networking/router/component.go

```diff
@@ -61,6 +61,20 @@ var RouterComponent = Component{
 					"[sig-networking] should not see excessive FailedToUpdateEndpointSlices Error updating Endpoint Slices",
 				},
 			},
+			{Suite: "Network_Edge"},
```

Comment:
Please update to `Network_Edge Component_Router` here since `Network_Edge` includes multiple components. Thanks